pennylane_lightning/lightning_kokkos/_adjoint_jacobian.py

```diff
@@ -300,11 +300,13 @@ def calculate_vjp(self, tape: QuantumTape, grad_vec):
             The vector-Jacobian products of a tape.
         """
 
+        # print(f'FDX01 : {grad_vec} : {qml.math.ndim(grad_vec)}')
```

Comment:
Do we want to keep this comment?
```suggestion
```

Reply:
These lines are not necessary. I will remove it :+1: